Remotion/Web/Core/UI/Controls/LazyEvaluatedValidationMessageControl.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Web.UI;
+using System.Web.UI.HtmlControls;
+using Remotion.Utilities;
+
+namespace Remotion.Web.UI.Controls
+{
+  public class LazyEvaluatedValidationMessageControl : HtmlContainerControl
+  {
+    private readonly IValidator _validator;
+
+    /// <summary>Initializes a new instance of the <see cref="LazyEvaluatedValidationMessageControl" /> class using the specified tag name and validator.</summary>
+    /// <param name="tag">A string that specifies the tag name of the control. </param>
+    /// <param name="validator">The validator for which the error message should be displayed.</param>
+    public LazyEvaluatedValidationMessageControl (string tag, IValidator validator)
+        : base(tag)
+    {
+      ArgumentUtility.CheckNotNull(nameof(validator), validator);
+      ArgumentUtility.CheckNotNull(nameof(tag), tag);
```

Comment:
Confusing when the arg check order does not match the parameter order :D

```suggestion
      ArgumentUtility.CheckNotNull(nameof(tag), tag);
      ArgumentUtility.CheckNotNull(nameof(validator), validator);
```